roles/spark/README.md

```diff
@@ -16,6 +16,8 @@ graph processing.
   `/usr/local/lib/libmesos.so`).
 - `zookeeper_master` - Hostname of Zookeeper master (default:
   `zookeeper.service.consul`).
+- `spark_log4j_host` - Hostname for logstash server (default: localhost)
```

Comment:
It would be great to keep all the options in a consistent style :)  Please add `` around default values and dots at the end.  If you don't like it, please change all other options instead :)